.github/workflows/run.yml

```diff
@@ -2,11 +2,23 @@ name: "Run OpenTelemetry demo"
 
 on:
   workflow_dispatch:
+  schedule:
+    - cron: '0 */4 * * *' # Run every 4 hours, at :00 sharp.
 
 jobs:
   run:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - name: Run
-        run: docker compose up --no-build
+        run:  sed -i "s/DEMO_HS_TOKEN/${DEMO_HS_TOKEN}/" './src/otelcollector/otelcol-config.yml' && docker compose build otelcol && docker compose up -d --no-build
+        env:
+          DEMO_HS_TOKEN: ${{ secrets.DEMO_HS_TOKEN }}
+
+      - name: Sleep for 5 minutes
```

Comment:
what happens during these 5 mins? can we control this differently?
aren't there scripts which trigger the different endpoints here?

Reply:
There's a load generator service as part of the docker compose - so we simply wait for it to generate data

Reply:
OOKKKKKK :))

Reply:
but linter is failing so let's fix it